Modules/posixmodule.c

```diff
@@ -216,6 +218,19 @@ extern char        *ctermid_r(char *);
 
 #endif /* !_MSC_VER */
 
+#if defined(__VXWORKS__)
+#include <vxCpuLib.h>
+#include <rtpLib.h>
+#include <wait.h>
+#include <taskLib.h>
+extern char ** environ;
+#ifndef _P_WAIT
+#define _P_WAIT          3
+#define _P_NOWAIT        2
+#define _P_NOWAITO       1
```

Comment:
Is there any reason to define these constants in a way different from what's normally done for POSIX, given that they aren't actually used in `rtpSpawn`?
https://github.com/python/cpython/blob/8589f14bbe55fb1ff5c765af5281100c38f0df63/Lib/os.py#L829-L830

Reply:
VxWorks RTOS has no definition for these macros like Windows. So we define them here as they will be used used later in this file.
`if (PyModule_AddIntConstant(m, "P_WAIT", _P_WAIT)) return -1;`